src/templates/partials/fold-karen-williams.hbs

```diff
@@ -107,7 +107,8 @@
       </li>
 
     </ul>
-    <img class="background" src="/assets/img/landing/karen-williams/background-overlay.png">
+
+    <div class="background"></div>
```

Comment:
Not using this intense `background-overlay.png` has perceivably improved the smoothness at which one scrolls over the Karen Williams section. It's hard work for browsers to blend transparent images onto the images and HTML behind it. It does that blending easier with CSS gradients.